src/main/java/org/maxgamer/quickshop/util/MsgUtil.java

```diff
@@ -314,11 +314,11 @@ public static void loadI18nFile() throws InvalidConfigurationException {
         /* Set default language vesion and update messages.yml */
         int ver = 0;
         String strVer = messagei18n.getString("language-version");
-        if (StringUtils.isNumeric(strVer)) {
-            try {
-                ver = Integer.parseInt(strVer);
-            } catch (NumberFormatException ignore) {
-            }
+
+        if (StringUtils.isNumeric(strVer) && !StringUtils.isEmpty(strVer)) {
+            ver = Integer.parseInt(strVer);
+        } else {
+            ver = messagei18n.getInt("language-version");
```

Comment:
Look like we should set `ver` to zero here or do nothing, it will cause NumberFormatException by 100%